core/primitives/src/types.rs

```diff
@@ -529,7 +529,12 @@ pub enum ValidatorKickoutReason {
     /// Validator unstaked themselves.
     Unstaked,
     /// Validator stake is now below threshold
-    NotEnoughStake { stake: Balance, threshold: Balance },
+    NotEnoughStake {
+        #[serde(with = "u128_dec_format", rename = "stake_u128")]
+        stake: Balance,
+        #[serde(with = "u128_dec_format", rename = "threshold_u128")]
```

Comment:
Actually why do we rename it?

Reply:
We currently have many u64 numbers exposed as JSON numbers (that is incorrect), and to avoid collision and keep backwards compatibility I am going to leave all the current number-fields and create “duplicates”, and the suffix states cleanly the possible range of values, which is useful for reliable clients implementation